test/extensions/filters/http/ratelimit/ratelimit_test.cc

```diff
@@ -766,7 +833,8 @@ TEST_F(HttpRateLimitFilterTest, ExcludeVirtualHost) {
                               _))
       .WillOnce(
           WithArgs<0>(Invoke([&](Filters::Common::RateLimit::RequestCallbacks& callbacks) -> void {
-            callbacks.complete(Filters::Common::RateLimit::LimitStatus::OK, nullptr, nullptr);
+            callbacks.complete(Filters::Common::RateLimit::LimitStatus::OK, nullptr, nullptr,
+                               "");
```

Comment:
There are (un-rebased) upstream changes to this test that add a 4th argument on a newline, so I made a similar style change here to help with that rebase in the future.